.gitignore

```diff
@@ -6,5 +6,6 @@
 /pkg/
 /spec/reports/
 /tmp/
+/vendor/
```

Comment:
Why this?

Reply:
I didn't want to commit it along with code changes. Is there a reason to remove this?

Reply:
You can always use `.git/info/exclude` if you need specific stuff to be ignored.